src/main/kotlin/io/github/thewisenerd/linters/sidekt/rules/JsonIgnorePropertiesOnDataClass.kt

```diff
@@ -0,0 +1,88 @@
+package io.github.thewisenerd.linters.sidekt.rules
+
+import io.github.thewisenerd.linters.sidekt.helpers.Debugger
+import io.gitlab.arturbosch.detekt.api.*
+import org.jetbrains.kotlin.psi.KtClass
+
+/**
+ * This rule enforces the use of the `@JsonIgnoreProperties` annotation on all data classes within the codebase.
+ *
+ * Severity: Maintainability
+ * Debt: 5min
+ *
+ * Usage:
+ * JsonIgnorePropertiesOnDataClass:
```

Comment:
add this brief to the main README?

Reply:
Yes, thanks for pointing out. Have added now